src/frontend/screens/Library/components/GameCard/index.tsx

```diff
@@ -291,7 +291,8 @@ const GameCard = ({
       // install
       label: t('button.install'),
       onclick: () => buttonClick(),
-      show: !isInstalled && !isQueued
+      show:
+        (!isInstalled && !isQueued) || (runner === 'sideload' && !isInstalled)
```

Comment:
Not sure what this is doing, `isQueued` is always `false` for sideloaded games, so that first condition also covers them

Reply:
actually, those are different things if you look closely.
First part is for non-sideloaded games:
- The game is installed
- The game is queued. 
So we dont show the install button on these cases.

The second part is for sideloading games that are not available
- Runner is sideload
- Is not installed

Reply:
True, just sent a fix 👍🏽